src/test/unit/marker-schema.test.js

```diff
@@ -235,12 +257,24 @@ describe('marker schema formatting', function () {
       ['percentage', 0.123456789],
       ['percentage', 1234.56789],
       ['percentage', 0.000123456],
+      ['unique-string', 0],
+      ['unique-string', 1],
```

Comment:
nit: It could be good to add a comment on the right side of them to explaining which string they refer to.